Antidote/AbstractCallViewController.h

```diff
@@ -8,31 +8,45 @@
 
 #import <UIKit/UIKit.h>
 
+#import "OCTToxAVConstants.h"
+
 @class OCTSubmanagerCalls;
 @class OCTChat;
 @class OCTCall;
 
+@protocol AbstractCallViewControllerDelegate <NSObject>
+
+@required
+- (void)dismissCurrentCall;
+- (void)callAccept;
+- (BOOL)sendCallControl:(OCTToxAVCallControl)control error:(NSError **)error;
+
+- (void)otherCallAccept:(BOOL)accept;
+
+@property (nonatomic, assign) BOOL enableMicrophone;
+
+@end
```

Comment:
Do you think it would be nicer to have it send itself as a reference as well in the protocol?

So instead of 
`- (void)activeCallDeclineButtonPressed;`

We could do

`- (void)activeCallDeclineButtonPressed:(ActiveCallViewController *)controller;`